content/docs/introduction/faq.md

```diff
@@ -219,7 +216,9 @@ native 64 bit integers would (only) help if you need integer precision
 above 2<sup>53</sup> but below 2<sup>63</sup>. In principle, support
 for different sample value types (including some kind of big integer,
 supporting even more than 64 bit) could be implemented, but it is not
-a priority right now. Note that a counter, even if incremented
+a priority right now. 
+
+NOTE: **Note:** A counter, even if incremented
```

Comment:
I don't think we need to emphasise our asides this strongly.

Reply:
If they aren't actually notes I'd recommend rewriting them. It's a confusing pattern for folks. But YMMV. :)

Reply:
This is mainly my style I think, in places where it's really important to stand out it'll be a separate paragraph already (e.g. in the `rate` function).